uk.co.pcem_emulator.pcem.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: uk.co.pcem_emulator.pcem
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: pcem-run
+finish-args:
+  - --share=network
+  - --share=ipc
+  # PCem has issues on Wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  # CD drive and joystick access
+  - --device=all
+  - --persist=.pcem
+  - --filesystem=xdg-documents
+
+modules:
+  - name: wxWidgets
+    rm-configure: true
+    config-opts:
+      - --with-gtk=3
+      - --disable-sdltest
+      - --with-cxx=14
+      - --enable-stl
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.4/wxWidgets-3.1.4.tar.bz2
+        sha256: 3ca3a19a14b407d0cdda507a7930c2e84ae1c8e74f946e0144d2fa7d881f1a94
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} .
+          - autoconf -f -B build/autoconf_prepend-include
+
+  - name: pcem
+    rm-configure: true
+    config-opts:
+      - --enable-release-build
+      - --enable-networking
+      - --enable-alsa
+    sources:
+      - type: git
+        url: https://github.com/sarah-walker-pcem/pcem
+        commit: 13f53a25687be71ee3ee8482b1a90b1b0aa64fb6
```

Comment:
This commit has a tag, `v17`. If you add it, shallow clone will work.

Reply:
Done, thanks!